cluster/extensions/src/main/scala/com/lightbend/lagom/internal/cluster/projections/ProjectionRegistry.scala

```diff
@@ -76,9 +85,27 @@ private[lagom] class ProjectionRegistry(system: ActorSystem) {
 
   }
 
+  implicit val exCtx: ExecutionContext = system.dispatcher
+  implicit val timeout: Timeout        = Timeout(1.seconds)
+
   private[lagom] def getStatus(): Future[DesiredState] = {
-    implicit val exCtx: ExecutionContext = system.dispatcher
-    implicit val timeout: Timeout        = Timeout(1.seconds)
     (projectionRegistryRef ? GetDesiredState).mapTo[DesiredState]
   }
+
+  def stopWorker(projectionWorkerName: String): Future[Done] =
+    (projectionRegistryRef ? Stop(projectionWorkerName))
+      .mapTo[Done]
+
+  def stopAllWorkers(projectionName: String): Future[Done] =
+    (projectionRegistryRef ? GetDesiredState)
+      .mapTo[DesiredState]
+      .map { desiredState =>
+        desiredState.findProjection(projectionName) match {
+          case None             => throw new ProjectionNotFound(projectionName)
+          case Some(projection) => projection
+        }
+      }
+      .map(_.workers.map(worker => stopWorker(worker.name)))
```

Comment:
`_.workers.map(worker => stopWorker(worker.name)))` will return a collection of `Future[Done]`, but we are not waiting for the acknowledgement that they were all accepted/processed. 

Different then `stopWorker` we are here really doing a fire-and-forget.